### PR TITLE
Bump Spring API version

### PR DESCRIPTION
**Pull request by geoand**
Requires: https://github.com/quarkusio/quarkus-spring-api/pull/10

Relates to: #8081

### CODE REVIEW

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -173,9 +173,9 @@
         <sundr.version>0.19.1</sundr.version> <!-- this is to avoid annoying pop-up in eclipse about failure to init Velocity logging -->
         <flapdoodle.mongo.version>2.2.0</flapdoodle.mongo.version>
         <wiremock.version>2.24.1</wiremock.version>
-        <quarkus-spring-api.version>5.2.Final</quarkus-spring-api.version>
+        <quarkus-spring-api.version>5.3.Alpha1-SNAPSHOT</quarkus-spring-api.version>
```
Comment (gsmet) [inline bom/runtime/pom.xml]: How does this one get updated to 5.3? Spring Core is still 5.2 AFAICS.

File: bom/runtime/pom.xml [hunk 2]
```diff
@@ -173,9 +173,9 @@
...
         <quarkus-spring-data-api.version>2.1.Final</quarkus-spring-data-api.version>
-        <quarkus-spring-security-api.version>5.2.Final</quarkus-spring-security-api.version>
+        <quarkus-spring-security-api.version>5.3.Alpha1-SNAPSHOT</quarkus-spring-security-api.version>
```
Comment (gsmet) [inline bom/runtime/pom.xml]: Spring Security has a 5.3 but did you upgrade to it?

**Review by gsmet — DISMISSED** [review]
Asked some questions regarding the versioning.

**Comment by geoand** [comment]
@gsmet this is just a test PR at this phase.

When we do release, it will be with version `5.2.x`. I only used `5.3.x` because that is what `master` of the spring API has (which should probably be changed)

cc @aureamunoz

File: bom/runtime/pom.xml (hunk 2 above)
Comment (geoand) [reply bom/runtime/pom.xml]: No, this was just changed by mistake

File: bom/runtime/pom.xml (hunk 1 above)
Comment (geoand) [reply bom/runtime/pom.xml]: Yes, see my comment

**Comment by gsmet** [comment]
Yeah, so we should probably use the version we currently have and only bump our version when we update the Spring component version.

**Comment by aureamunoz** [comment]
`quarkus-spring-api 5.2.SP1` has been released : https://repo.maven.apache.org/maven2/io/quarkus/quarkus-spring-api/5.2.SP1/

File: bom/runtime/pom.xml (hunk 1 above)
Comment (geoand) [reply bom/runtime/pom.xml]: Fixed

File: bom/runtime/pom.xml (hunk 2 above)
Comment (geoand) [reply bom/runtime/pom.xml]: Fixed

**Comment by geoand** [comment]
@gsmet OK to merge?

**Review by machi1990 — APPROVED** [review]

**Pull request merged**